R/data.table.R

```diff
@@ -240,8 +240,7 @@ chmatch2 <- function(x, table, nomatch=NA_integer_) {
     return(ans)
   }
   if (!mult %chin% c("first","last","all")) stop("mult argument can only be 'first','last' or 'all'")
-  missingroll = missing(roll)
```

Comment:
these arguments have defaults, shouldn't be possible to be missing; `missingwith` is unused anyway

Reply:
default value still is missing if not provided explicitly.
```
> (function(a=1) print(missing(a)))()
[1] TRUE
> (function(a=1) print(missing(a)))(1)
[1] FALSE
```